components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -249,12 +251,37 @@ message WorkspaceInstanceStatus {
     // Admission describes who can access a workspace instance and its ports.
     AdmissionLevel admission = 6;
 
+    // ports is the list of served ports in the workspace.
+    repeated Port ports = 7;
```

Comment:
There's a bit of an assymetry here.

`ControlPort(ControlPortRequest)` is identified by Workspace ID, but the resulting `ports` are shown on the `Workspace Instance`. What limits us from putting the Ports on the WorkspaceStatus rather than the WorkspaceInstanceStatus?